doc/getting-started.rst

```diff
@@ -166,41 +174,68 @@ Now you can build and re-run your code to see the new output:
         /    /  /    /   \    \
        /____/  /____/     \____\
 
-Run a single-file Haskell script
---------------------------------
+Running a single-file Haskell script
+------------------------------------
 
-Cabal also enables us to run single-file Haskell scripts
-without creating a project directory or ``.cabal`` file.
-The cabal directives are placed in the file within a comment.
+Cabal also supports running a single self-contained Haskell script like 
+the following file named ``myscript``:
 
 .. code-block:: haskell
-
+    
     #!/usr/bin/env cabal
     {- cabal:
-    build-depends: base, split
+    build-depends:
+      base ^>=4.19.0.0,
+      haskell-say ^>=1.0.0.0
     -}
 
-    import Data.List.Split (chunksOf)
+    import HaskellSay (haskellSay)
 
     main :: IO ()
-    main = getLine >>= print . chunksOf 3
+    main = haskellSay "Hello, Haskell!"
 
-This can be run using ``cabal run myscript``.
-On Unix-like systems this can be run directly with execute permission.
+The necessary sections of a ``.cabal`` file are placed
+directly into the script as a comment.
+
+To execute this script, run:
 
 .. code-block:: console
 
     $ cabal run myscript
 
+On Unix-like systems this can be run directly with execute permission.
+
+.. code-block:: console
+
     $ chmod +x myscript
     $ ./myscript
+       ________________________________________________________
+      /                                                        \
+     | Hello, Haskell!                                          |
+      \____       _____________________________________________/
+           \    /
+            \  /
+             \/
+       _____   _____
+       \    \  \    \
+        \    \  \    \
+         \    \  \    \
+          \    \  \    \  \-----------|
+           \    \  \    \  \          |
+            \    \  \    \  \---------|
+            /    /  /     \
+           /    /  /       \  \-------|
+          /    /  /    ^    \  \      |
+         /    /  /    / \    \  \ ----|
+        /    /  /    /   \    \
+       /____/  /____/     \____\
 
 Project metadata can also be included:
```

Comment:
Let's ditch this detail? I think it's enough for a getting started session and all rest should be read in the respective manual section (handily referenced below).

Reply:
Removed